src/components/displayAll.js

```diff
@@ -0,0 +1,54 @@
+import React, { useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { useSelector, useDispatch } from 'react-redux';
+
+import { FetchStockData, ShowStockData } from '../redux/Reducer';
+
+function DisplayAll() {
+  const Stocks = useSelector((state) => state.stocks);
+  const dispatch = useDispatch();
+
+  const Navigate = useNavigate();
+  useEffect(() => {
+    dispatch(FetchStockData());
+  }, []);
+
+  const obtainDetails = (symbol) => {
+    dispatch(ShowStockData(symbol));
+  };
+
+  return (
+    <>
+
+      <h3 className="main-heading"> Crypto currency market data</h3>
+
+      <div className="information">
+
+        {
+
+Stocks.length > 1
+  ? Stocks.map((stock) => (
+    <div key={stock.symbol} className="columnDiv">
+      <button
+        className=""
```

Comment:
- [ ] it would be better if you add  filter to filter the data as required in order to complete the requirements 
- Screenshot of the requirements 
<img width="1069" alt="Screenshot 2022-11-30 at 17 13 54" src="https://user-images.githubusercontent.com/61790587/205037189-26a96578-3518-4223-9548-15785144a1c0.png">